.travis.yml

```diff
@@ -4,10 +4,21 @@ sudo: false
 language: objective-c
 
 env:
-  - SDK="iphoneos12.4"
+  global:
+    - PROJECT="Toaster.xcodeproj"
+    - SCHEME="Toaster"
+    - SDK="iphoneos12.4"
+  matrix:
+    - DESTINATION="id=3A381AFE-A894-425F-A503-92C84DA4086A"
```

Comment:
Let's avoid simulator identifiers in this file.

Instead, two alternatives:
- using `DESTINATION_UDID=$(instruments -s devices | grep "$DEVICE \[" | sed -E 's/.*\[([0-9A-F-]+)\].*/\1/g');` as I wrote in https://github.com/ZipArchive/ZipArchive/blob/master/.travis.yml
- using `xcode_destination` as I wrote in https://github.com/magicalpanda/MagicalRecord/blob/master/.travis.yml

Reply:
Thanks! I will change it.